appeals/api/src/server/endpoints/integrations/integrations.validators.js

```diff
@@ -0,0 +1,36 @@
+import { readdirSync, readFileSync } from 'node:fs';
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
+
+const SCHEMA_PATH = './src/message-schemas';
+const loadSchemas = async () => {
```

Comment:
It might be worth caching the loaded schemas, else we'll be reading the files each time a request comes in.